app/models/events/review.rb

```diff
@@ -35,5 +37,29 @@ class Review < ApplicationRecord
 
     validates :status, inclusion: { in: statuses.keys }
     validates :github_id, presence: true
+
+    ##
+    # Returns the occurrence time of the 'review' event.
+    ##
+    def occurence_time(payload:)
+      event_action = payload.fetch('action')
+      review_payload = payload.fetch('review')
+      case event_action
+      when 'submitted'
+        parse_time(review_payload.fetch('submitted_at'))
+      when 'edited'
+        parse_time(review_payload.fetch('edited_at'))
+      when 'dismissed'
+        parse_time(review_payload.fetch('dismissed_at'))
+      else
+        raise "Uknown action '#{event_action}'"
```

Comment:
I believe that this a GitHub logic, I would take it off from the model and put it in a service

Reply:
Done it. Thanks!